projects/components/src/datagrid/directives/select-directive.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { OnChanges } from '@angular/core';
+import { EventEmitter } from '@angular/core';
+import { Directive, Input } from '@angular/core';
+import { ClrDatagrid } from '@clr/angular';
+import { Host } from '@angular/core';
+import { GridSelectionType } from '../datagrid.component';
+
+@Directive({
+    selector: '[vcdComponentGridSelection]',
```

Comment:
This is only meant to be used on vcd grid, right? That should be part of the selector so you can't use it anywhere else and it's also clearer